diagnostic.md

```diff
@@ -15,14 +17,53 @@ Place your responses inside the fenced code-blocks where indivated by comments.
   join table with references to `Movies` and `Profiles`.
 
   ```md
-    # < Your Response Here >
+  create_table "Profiles", force: :cascade do |t|
+    t.string   "given_name",  null: false
+    t.string   "surname", null: false
+    t.string   "email"
+    t.datetime "created_at",  null: false
+    t.datetime "updated_at",  null: false
+  end
+    create_table "Movies", force: :cascade do |t|
+      t.string   "title",  null: false
+      t.string   "release_date", null: false
+      t.string   "length"
+      t.datetime "created_at",  null: false
+      t.datetime "updated_at",  null: false
+  end
+      create_table "Favorites", force: :cascade do |t|
+        t.string   "given_name",  null: false
+        t.string   "surname", null: false
+        t.string   "email"
+        t.datetime "created_at",  null: false
+        t.datetime "updated_at",  null: false
+  end
+
+add_foreign_key "favorites", "profiles"
+add_foreign_key "favorites", "movies"
+add_foreign_key "profiles", "movies"
+  end
   ```
 
 1.  For the above example, what needs to be added to the Model files?
 
   ```rb
   class Profile < ActiveRecord::Base
+
```

Comment:
Don't forget to specifically define each `class`. Also, the profile class should look something like this:
```
class Profile < ActiveRecord::Base
  has_many :movies, through: :favorites
  has_many :favorites
end
```